libs/partners/upstage/langchain_upstage/embeddings.py

```diff
@@ -48,9 +51,9 @@ class UpstageEmbeddings(BaseModel, Embeddings):
 
     client: Any = Field(default=None, exclude=True)  #: :meta private:
     async_client: Any = Field(default=None, exclude=True)  #: :meta private:
-    model: str = "solar-1-mini-embedding"
+    model: str = Field(default=None)
```

Comment:
I don't think you want this. Should we update the test to pass one of the available models instead?

Reply:
With this implementation, you'll be able to create `UpstageEmbeddings()` but when you call it, it'll try to call the model `None-passage` which users probably don't want

Reply:
Yes right. I think this Field should be  `Field(...)`